### PR TITLE
Update Models

### PR DESCRIPTION
**Pull request by inishchith**
- Add Smtp Port and Host for sender under Models 
- Prettify Email List [ Have to Update later ]

### CODE REVIEW

File: email/sender.js [hunk 1]
```diff
@@ -1,43 +1,43 @@
...
-          clientSecret:config.clientSecret_input,
-          refreshToken:config.refreshToken_input    }
-        
-    } 
-);
+      type: 'OAuth2',
+      user: config.userMail,
+      clientId: config.clientId_input,
+      clientSecret: config.clientSecret_input,
+      refreshToken: config.refreshToken_input
```
Comment (inishchith) [inline email/sender.js:19]: @accakks Do we need [17-19 ] anymore ?

Comment (accakks) [reply email/sender.js:19]: we need userMail and Password right?

Comment (inishchith) [reply email/sender.js:19]: yes . just that

**Comment by inishchith** [comment]
#close

**Pull request closed without merging**